src/sagemaker/jumpstart/model.py

```diff
@@ -1042,6 +1058,7 @@ def _get_deployment_configs(
                 region=self.region,
                 model_version=self.model_version,
                 hub_arn=self.hub_arn,
+                accept_draft_model_eula=True,
```

Comment:
Why is this hardcoded ? This needs to come from user input.

On a separate note, why is there no `accept_eula` for the JumpStart target model ? How does this method work for Gated JS models ?

Reply:
Yea so this is hardcoded because the `display_benchmark_metrics()` calls the jumpstart model `get_init_kwargs`. So the customer wouldn't even be able to see the various combinations without getting an accept eula error.

Cases where this blocks the CX:
1. What if the customer does not want a deployment config with a gated draft model?
2. What if there are different deployment configs with different eulas

IMO hardcoding here is okay because ultimately they will have to ack the EULA anyways when they actually pick the deployment config they want. This is just to enable displaying of the deployment configs + metrics without any blockers

Reply:
Regarding accept_eula of the target model, this occurs at time of deployment through `Model.deploy()` as opposed to model construction which I think we originally wanted

Reply:
Ack. Please add test cases to make sure this expectation is upheld. We do not want customers to be able to deploy models when they did not explicitly `accept_eula`